db/patch-http-body-encoding.sql

```diff
@@ -0,0 +1,6 @@
+-- You should not modify if this have pushed to Github, unless it does serious wrong with the db.
+BEGIN TRANSACTION;
+
+ALTER TABLE [monitor] ADD http_body_encoding TEXT;
```

Comment:
VARCHAR(20) should be enough?

Reply:
Fixed. But I configure it to 25 characters because "x-www-form-urlencoded" can be added here in the future.